community/detectors/InfluxDB_cve_2019_20933/README.md

```diff
@@ -0,0 +1,16 @@
+# CVE-2019-20933 VulnDetector
+
+InfluxDB before 1.7.6 has an authentication bypass vulnerability in the authenticate function in
```

Comment:
Could you also add a section about how InfluxDB doesn't enable authentication by default, and this plugin also checks for InfluxDB with no authentication.

Also, to conform with the directory naming convention, could you lowercase the detector folder name to "influxdb_cve_2019_20933"?

Reply:
OK I improved the README and change the folder name to lowercase.

Reply:
The update looks great, thanks for the fast turn-around. I've applied a few minor changes to your change set directly to pass the internal check. I simplified the db query a bit by dropping the "db=" part, it doesn't seem to be needed for the "show database" query after testing against the container images you provided.